source/install-desktops/enable-reverse-proxy.rst

```diff
@@ -0,0 +1,149 @@
+.. _install-desktops-enable-reverse-proxy:
+
+Enable Reverse Proxy
+====================
+
+The reverse proxy will proxy a request to any specified host and port through
+IP sockets. This is different than what is used for proxying to per-user NGINX
+processes through Unix domain sockets. This can be used to connect to Jupyter
+notebook servers, RStudio servers, VNC servers, and more... This is disabled by
+default as it can be a security risk if not properly setup using a good
+``host_regex``.
```

Comment:
After this summary, a short explanation would be a good idea with examples of both node and rnode i.e. here is a really rough draft:

 when enabled making a get request to /node/host123/5321/foo/bar will proxy to a server running on host123 and port 5321, making a get request to that server with the URL /node/host123/5321/foo/bar. Making a get request ti /rnode/host123/5321/foo/bar will proxy to a server running on host123 and port 5321, making a get reqeust to that server with the URL /foo/bar. Thus it is named "rnode" because when accessing a server through /rnode/ any links the server produces in its response (to assets or other pages, etc.) would need to be relative to the server root. Any server responding to requests "node" would need to be configured to run at the suburi /node/host123/5321. noVNC is configured to use /rnode/, while Jupyter and RStudio servers would be configured to use /node/.